src/pudl_archiver/__init__.py

```diff
@@ -27,7 +29,18 @@ def all_archivers():
     for module in module_names:
         # AbstractDatasetArchiver won't know about the subclasses unless they are imported
         __import__(module)
-    return AbstractDatasetArchiver.__subclasses__()
+
+    def all_subclasses(cls):
```

Comment:
@zschira This is the only part of the transition to subclassing the NGQV archiver that felt a bit questionable, but otherwise we wind up having a non-functional NGQV archiver class in the list and not including 191 or 757A. If you have suggestions on how to do this better I'm all ears.

Reply:
Hmm yeah it's a bit annoying, but I might just leave it. We could pull out the `get_datasets` and `get_year_resource` methods to be standalone functions that get called by the archivers, but I'm not sure if it's actually worth it